tensorflow/compiler/xla/client/lib/qr_test.cc

```diff
@@ -33,6 +34,7 @@ namespace {
 using QrTest = xla::ClientLibraryTestBase;
 
 XLA_TEST_F(QrTest, Simple) {
+  tensorflow::allow_tf32_execution(false);
```

Comment:
Let's leave some breadcrumbs here on what's going on, maybe wrap `allow_tf32_execution` into a function that has a comment explaining what's going on.

Reply:
Added the comment "Test fails with tf32 allowed". I'm not sure what specific part of the test requires full precision, but I strongly suspect it is the `xla::BatchDot`. I don't think creating a wrapper function is worth it.